armada/handlers/k8s.py

```diff
@@ -58,3 +58,10 @@ def get_namespace_pod(self, namespace="default"):
         '''
 
         return self.client.list_namespaced_pod(namespace)
+
+    def get_all_pods(self):
+        '''
```

Comment:
I can change the voice of the description but all other methods in this file are annotated with ''' not """ and I think it should be consistent